docs/content/intro/quickstart.md

```diff
@@ -0,0 +1,314 @@
+- The second one creates a Project that has no dependency on a Version Control System and its Builds will be triggered via POST requests on Brigade's [Generic Gateway](https://docs.brigade.sh/topics/genericgateway/). Think of this approach as some JavaScript code that will do stuff with containers and be triggered by a POST message (either a plain JSON one or a [CloudEvent](https://cloudevents.io/)).
+
+By the way, this does not mean that you cannot combine these scenarios (i.e. have your Builds triggered by POST requests and your source code be pulled and acted upon) or think of alternative ways to use Brigade!
+
+## Using Brigade with a Version Control System
+
+### Creating A New Project - using a Version Control System
+
+To create a new project, use `brig project create` and answer the prompts. Feel free to modify or leave all options at their defaults (just press Enter on every interactive prompt).
+
+```bash
+brig project create
+```
+
+Output would be similar to this:
+```
+? Project name deis/empty-testbed
+? Full repository name github.com/deis/empty-testbed
+? Clone URL (https://github.com/your/repo.git) https://github.com/deis/empty-testbed.git
+? Add secrets? No
+Auto-generated a Shared Secret: "novxKi64FKWyvU4EPZulyo0o"
+? Configure GitHub Access? No
+? Configure advanced options No
+Project ID: brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac
+```
+
+Here we're using the name 'deis/empty-testbed' for our project, which points to a test repo on 'https://github.com/deis/empty-testbed'. Of course, don't forget to give a proper name to your project, as well as set the 'Clone URL' correctly. If it's wrong, your subsequent Builds will fail! For documentation on project creation, check [here](https://docs.brigade.sh/topics/projects/).
+
+Now we can view the newly created project:
+```bash
+brig project list
+```
+
+Output would be something like:
+```
+NAME                    ID                                                              REPO
+myusername/myproject    brigade-2e9bb93bf149536a951d236772ae8be77a3cef9335c82bf39fc18c  github.com/myusername/myproject
+```
+
+You can also do a `kubectl get secret` to view the [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/) that was created for this Project. Bear in mind that Brigade stores information about its entities (Project/Build) in Secrets.
+
+```
+NAME                                                             TYPE                                  DATA   AGE
+brigade-2e9bb93bf149536a951d236772ae8be77a3cef9335c82bf39fc18c   brigade.sh/project                    24     2m
+...other cluster Secrets...
+```
+
+### Creating Your First Brigade Build
+
+Conceptually, a Brigade Build is a set of instructions (written in JavaScript, in a file called `brigade.js`) that will run upon a triggering of an event. Events can be something like a git push, a Docker push or just a webhook. In this way, you can think of Builds as jobs/tasks/pipelines.
```

Comment:
I wonder if it is worth mentioning that the designated test repo (`brigadecore/empty-tesbed`) has its own [brigade.js](https://github.com/brigadecore/empty-testbed/blob/master/brigade.js) file existing on the master branch and that this would be used if the user runs `brig run brigadecore/empty-testbed` without any local `brigade.js` file supplied (as is below)?

(Technically, that brigade.js file doesn't yet have a handler for the `exec` event, so it'll be a no-op...)

I guess, more generally, perhaps making it clear that the `brigade.js` file existing in the vcs repo as set up on the project will be used if none is supplied.

Reply:
fixed, let me know :)

Reply:
👍 Perfect!  Thanks.